stable/elasticsearch-exporter/.helmignore

```diff
@@ -0,0 +1,3 @@
+.git
```

Comment:
Do you want to go ahead and add the other common patterns from `helm create`?

Reply:
Done!